### ISSUE TITLE
Nom des champs et doublon

### ISSUE THREAD

**Issue by nicolas-bonnel** [cget-carto/mission_coworking#14]
Bonjour,

Certains champs sont en double, comme INSEECOM. Serait il possible d'avoir la liste des libellé pour chaque champ, ce qui permettrait de lever l'ambiguité à certains endroits ?

Merci

**Comment by bzg** [comment]
Oui, ces propositions sont implémentées dans les deux PR du dépôt. Il reste à les fusionner avec quelques ajustements manuels. À disposition pour aider si besoin.
